File: C++/FactorialDeUnNúmero.cpp
#include <iostream>
using namespace std;

int main()
{

    int ingresaNumero;
    int facotiral;

    cout << "Cual es el factorial de: ";
    cin >> ingresaNumero;
    cout << "El factorial de " << ingresaNumero << " es ";

    for (int i = 1; i <= ingresaNumero; i++)
        facotiral *= i;
    {
        cout << facotiral;
    }

    return 0;
}

File: JavaScript/FactorialDeUnNúmero.js
let ingresaNumero = parseInt(prompt("Ingresa el factorial que deseas saber: "))
let factorial = 1

for (let index = 1; index <= ingresaNumero; index++) {
    factorial *= index
    console.log(`El factorial de ${ingresaNumero} es: ${factorial}`)
}